python_modules/dagit/dagit/schema.py

```diff
@@ -93,14 +218,23 @@ def resolve_types(self, info, pipelineName):
 class Pipeline(graphene.ObjectType):
     name = graphene.NonNull(graphene.String)
     description = graphene.String()
-    solids = graphene.NonNull(graphene.List(lambda: graphene.NonNull(Solid)))
-    contexts = graphene.NonNull(graphene.List(lambda: graphene.NonNull(PipelineContext)))
+    solids = non_null_list(lambda: Solid)
+    contexts = non_null_list(lambda: PipelineContext)
     environment_type = graphene.NonNull(lambda: Type)
+    computeNodeGraph = graphene.Field(
+        graphene.NonNull(lambda: ComputeNodeGraph),
+        args={
+            'config': graphene.Argument(graphene.NonNull(graphene.String)),
```

Comment:
I've been thinking about that after our call and I'm not 100% sure config should be passed as string. However it's fine for this moment.

Should this be optional though? Is empty config a valid config?